packages/grid/src/GridModel.ts

```diff
@@ -71,6 +72,10 @@ abstract class GridModel<
    */
   abstract textForCell(column: ModelIndex, row: ModelIndex): string;
 
+  tooltipForCell(column: GridRangeIndex, row: GridRangeIndex): string | null {
+    return null;
+  }
```

Comment:
Add to `IrisGridModel` instead, this isn't actually used within `Grid` (and it doesn't display tooltips).